resources.whatwg.org/build/deploy.sh

```diff
@@ -181,9 +181,9 @@ if [[ "$TRAVIS" == "true" ]]; then
     curlretry --fail --remote-name --location https://github.com/validator/validator/releases/download/linux/vnu.linux.zip
     unzip vnu.linux.zip
```

Comment:
While here maybe find a way to make unzip not list everything it unzips? That has felt like noise in the last few logs I've seen.

Reply:
> While here maybe find a way to make unzip not list everything it unzips? That has felt like noise in the last few logs I've seen.

Yeah, agreed — added the `-q` option, to suppress the list of everything it unzips